### PR TITLE
Jhia 61 forgotten password screen

### PR DESCRIPTION
**Pull request by Joshw8**
Contains work for the LoginScreen and ForgottenPassword flow screens

### CODE REVIEW

File: src/types/forgotPassword.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { NativeStackNavigationProp } from '@react-navigation/native-stack'
+import { RootStackParamList } from '@/types/navigator'
+
```
Comment (danielj24) [inline src/types/forgotPassword.ts:3]: Quite a few re-usable types in this file, can we define them once and re-use please?

File: src/screens/ForgottenPasswordScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      locations={[0, 0.6]}
+      className="w-full h-full px-5 pt-20 pb-12"
+      style={{ flex: 1 }}
+    >
+      <View className="flex flex-col justify-between h-full items">
+        <AuthHeader />
+
+        <View className="flex flex-col items-center justify-center"></View>
+
+        {step === 0 ? (
```
Comment (danielj24) [inline src/screens/ForgottenPasswordScreen.tsx:35]: If we go with the renaming of steps. We can probably do it via an enum and switch?

``` typescript
switch (step) {
    case Steps.SEND_EMAIL:
      return StepSendEmail // component
}
```

**Review by danielj24 — CHANGES_REQUESTED** [review]

File: src/types/forgotPassword.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export interface Step1Props {
```
[the remark this answers is not in the document]
Comment (danielj24) [reply src/types/forgotPassword.ts:4]: `StepResetConfirmation` instead of `StepPasswordReset`

File: src/screens/ForgottenPasswordScreen.tsx (hunk 2 above)
Comment (Joshw8) [reply src/screens/ForgottenPasswordScreen.tsx]: 👍

File: src/types/forgotPassword.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Joshw8) [reply src/types/forgotPassword.ts]: 👍

**Review by danielj24 — APPROVED** [review]

**Pull request merged**